setup.py

```diff
@@ -132,6 +133,7 @@
             'cap_fixtures = cap.modules.fixtures.ext:CAPFixtures',
             'cap_xrootd = cap.modules.xrootd.ext:CapXRootD',
             'cap_auth = cap.modules.auth.ext:CAPOAuth',
+            'cap_mail = cap.modules.mail.ext:CAPMail',
```

Comment:
Not needed.. Initialized from https://github.com/inveniosoftware/invenio-mail/blob/master/setup.py#L79